tools/crm_resource_runtime.c

```diff
@@ -1899,7 +1899,8 @@ wait_till_stable(pcmk__output_t *out, int timeout_ms, cib_t * cib)
             }
         }
 
-    } while (actions_are_pending(scheduler->actions));
+        search = xpath_search(scheduler->input, "/" XML_LRM_TAG_RSC_OP "[@" XML_LRM_ATTR_RC "=='193']");
```

Comment:
this would work with "//" to do a deep search, but it's more efficient to use "/" with each element stepping down into it